### ISSUE TITLE
make test fails with default project

### ISSUE THREAD

**Issue by greenkiwi** [aio-libs/create-aio-app#171]
I have created a sample application `test_example` and then when I run the tests, I get a failure `ERROR: file not found: make`

Create application
```
create-aio-app test-example --without-postgres
cd test_example
make test
```

Output
```
WARNING: Image for service test was built because it did not already exist. To rebuild this image you must use `docker-compose build` or `docker-compose up --build`.
Creating test_example_test ... done
Attaching to test_example_test
test_1              | ============================= test session starts ==============================
test_1              | platform linux -- Python 3.7.6, pytest-5.2.0, py-1.8.1, pluggy-0.13.1 -- /usr/local/bin/python
test_1              | cachedir: .pytest_cache
test_1              | rootdir: /app
test_1              | plugins: aiohttp-0.3.0, cov-2.6.1
collected 0 items
test_1              |
test_1              | ============================ no tests ran in 0.04s =============================
test_1              | ERROR: file not found: make
test_1              |
Stopping test_example_test ... done
```

Is this expected?

I would have expected to have the stock test `test_views.test_view` execute.

**Comment by Arfey** [comment]
hi, thx for ur report 😀

i fixed problem

https://github.com/aio-libs/create-aio-app/blob/88bf76809871bea733d5862f44a2a88237aedb49/create_aio_app/template/%7B%7Bcookiecutter.project_name%7D%7D/docker-compose.yml#L61

u need just add that to ur compose file.

Also i did a new release with fix `0.0.9`.
ps: if u'll use a new release with `--without-postgres` u should to see `'typing.AsyncGenerator' imported but unused` after run test. it's quick to fix, just remove this import. I'll fix it later ... or u can help me 😀

**Comment by greenkiwi** [comment]
@Arfey thx!!! 🥇
